dev/tests/integration/testsuite/Magento/Sales/_files/guest_quote_with_addresses.php

```diff
@@ -10,27 +10,31 @@
 \Magento\TestFramework\Helper\Bootstrap::getInstance()->loadArea(\Magento\Framework\App\Area::AREA_FRONTEND);
 
 $objectManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager();
-/** @var \Magento\Catalog\Model\Product $product */
-$product = $objectManager->create(\Magento\Catalog\Model\Product::class);
-$product->setTypeId('simple')
-    ->setAttributeSetId($product->getDefaultAttributeSetId())
-    ->setName('Simple Product')
-    ->setSku('simple-product-guest-quote')
-    ->setPrice(10)
-    ->setTaxClassId(0)
-    ->setMetaTitle('meta title')
-    ->setMetaKeyword('meta keyword')
-    ->setMetaDescription('meta description')
-    ->setVisibility(\Magento\Catalog\Model\Product\Visibility::VISIBILITY_BOTH)
-    ->setStatus(\Magento\Catalog\Model\Product\Attribute\Source\Status::STATUS_ENABLED)
-    ->setStockData(
-        [
-            'qty' => 100,
-            'is_in_stock' => 1,
-        ]
-    )->save();
-
 $productRepository = $objectManager->create(\Magento\Catalog\Api\ProductRepositoryInterface::class);
+
+try {
```

Comment:
Don't use try try-catch construction in fixtures. Every fixture should have a rollback and that's enough.
Short story: reinstall Magento, remove try-catch and run test again it is passing locally.

Long story: Sometimes(very often for me) when test exited with error people try to fix the problem and then they try again. On this step if they do not succeed they will have 2 equal records in the database. But rollback expects only one record... so don't use try-catch

Reply:
> Don't use try-catch construction in fixtures. Every fixture should have a rollback and that's enough.
> Short story: reinstall Magento, remove try-catch and run test again it is passing locally.
> 
> Long story: Sometimes(very often for me) when test exited with error people try to fix the problem and then they try again. On this step if they do not succeed they will have 2 equal records in the database. But rollback expects only one record... so don't use try-catch

Thank you for notice, @TomashKhamlai !

Working on solution...